contributing/index.md

```diff
@@ -1,45 +1,74 @@
 (sprint_help)=
-# Contribute
+# Contribute to the events
 
-% TODO: make nicer and add missing pages/sections
+:::{toctree}
+:maxdepth: 1
+:hidden:
 
-## ...with the website
-To build the website locally, install the requirements
+self
+../about/contributing_to_pymc/index
+../about/contributing_to_documentation/docs_presentation
+donate
+:::
+
+## Help us improve this website
+
+If you find any errors or have any suggestions, please let us know! You can either [report an issue](https://github.com/pymc-devs/pymc-data-umbrella/issues/new) or send us a pull request.
+
+1. Click in the GitHub icon in the top right corner of the page to visit the page repo.
 
-`pip install -r requirements-docs.txt`
+2. Fork and clone the repository.
 
-and then run sphinx:
+3. Install the requirements.
 
-`sphinx-build . build -b html`
+    ```bash
+    pip install -r requirements-docs.txt
+    ```
 
-## ...with outreach
-Share the announcements on social networks, within your personal networks...
+4. Build the docs locally.
 
-<a class="twitter-share-button"
-  href="https://twitter.com/intent/tweet?text=https://pymc-data-umbrella.xyz">
-Create Tweet</a>
+    ```bash
+    sphinx-build . build -b html
+    ```
+
+5. Make your changes to the documentation.
+
+6. Create your pull request.
+
+:::{tip}
+You can use as reference the {ref}`PR Tutorial <pr_tutorial>` which explains how you can do a PR to PyMC.
+:::
+
+## Spread the word
+
+Share the announcements on social networks, within your personal networks. This will help us to reach more people.
+
+<a href="https://twitter.com/intent/tweet?text=https://pymc-data-umbrella.xyz">
+  Create Tweet <i class="fab fa-twitter h4 text-main"></i>
+</a>
+
+### Some outreach related advise:
 
-Some outreach related advise:
 * Always use [https://pymc-data-umbrella.xyz](https://pymc-data-umbrella.xyz) when linking
   to the general series of events, use links within this domain for specific links to single
   events or external links (i.e. direct link to webinar or to registration form).
 
   **DO NOT USE** `pymc-data-umbrella.readthedocs.io` nor `sprint-pymc-data-umbrella.readhedocs.io`
   domains.
```

Comment:
```suggestion
```
@OriolAbril 
Can we remove this? I think it's more distracting that useful.  It is residual from the very initial website, which is not relevant anymore.

Reply:
yes, we can remove this

Reply:
Done!